src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AuthorizationApiServiceImpl.scala

```diff
@@ -90,7 +90,7 @@ final case class AuthorizationApiServiceImpl(
   private def assertTenantAllowed(selfcareId: String, origin: String): Future[Unit] =
     Future
       .failed(UnknownTenantOrigin(selfcareId))
-      .unlessA(origin == "IPA" || allowList.contains(selfcareId))
+      .unlessA(origin == "IPA" || origin == "ANAC" || allowList.contains(selfcareId))
```

Comment:
May I suggest to correct this with these two hints:
- avoid using `unlessA` because the cats features do not fit well with Future
- using a `Set.contains` instead of this OR expression